modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets_{context}"]
+= Creating a secret using the web console
+
+You can create secrets using the web console. 
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Click *Create* -> *From YAML*.
+.. Edit the YAML manually to your specifications, or drag and drop a file into the YAML editor. 
+For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+ name: example
+ namespace: <namespace>
+type: Opaque <1>
+data:
+stringData: <2>
+  username: <username>
+  password: <password>
+----
+<1> This example specifies an opaque secret; however, you may see other secret types such as service account token secret, basic authentication secret, SSH authentication secret, or a secret that uses Docker configuration. 
+<2> Specifies the basic authentication values to use.
```

Comment:
Is line 32 <2> intended for explaining `stringData: <2>`?  Line 32 does not explain usage of `stringData: <2>`. If intended for explaining `stringData: <2>`, you could refer to the statement `Entries in the stringData map are converted to base64 and the entry will then be moved to the data ...` under the `Understanding secrets` section.